murmur128.go

```diff
@@ -14,9 +14,9 @@ const (
 
 // Make sure interfaces are correctly implemented.
 var (
-	_ hash.Hash = new(digest128)
-	_ Hash128   = new(digest128)
-	_ bmixer    = new(digest128)
+	_ hash.Hash = new(Digest128)
+	_ Hash128   = new(Digest128)
+	_ bmixer    = new(Digest128)
```

Comment:
This is a more common pattern in Go:
```suggestion
	_ hash.Hash = (*Digest128)(nil)
	_ Hash128   = (*Digest128)(nil)
	_ bmixer    = (*Digest128)(nil)
```